### PR TITLE
Improved "what's next?" section

### PR DESCRIPTION
**Pull request by TheGhoulRe**
The `What's next?` section in the `Getting-Started` page was hard to follow. So, I made a some modifications

### CODE REVIEW

File: guides/Getting-Started.md [hunk 1]
```diff
@@ -340,8 +340,12 @@ The generated modules will be placed in the `output` directory by default. Each
...
 
-New PureScript programmers are also encouraged to spend some time browsing [Pursuit](https://pursuit.purescript.org), which hosts generated API documentation for PureScript libraries. In particular it is worth familiarising yourself with the [core libraries](https://github.com/purescript) (i.e., those which are hosted under the `purescript` organisation on GitHub), and especially the [prelude](https://pursuit.purescript.org/packages/purescript-prelude), as these provide many basic concepts which are frequently useful for writing programs.
+If you are somewhat familiar with typed functional programming, You may alternatively browse the [language reference in the documentation repository](https://github.com/purescript/documentation/tree/master/language). It gives a brief, reference-style description of the language.
+
+Haskell programmers will find the [Differences from Haskell](https://github.com/purescript/documentation/blob/master/language/Differences-from-Haskell.md) page very  useful.
```
Comment (JordanMartinez) [inline guides/Getting-Started.md:349]:
```diff
- page very  useful
+ page very useful
```

Comment (TheGhoulRe) [reply guides/Getting-Started.md]: Done

Comment (JordanMartinez) [reply guides/Getting-Started.md]: Thanks
